app/controllers/account_invitations_controller.rb

```diff
@@ -18,8 +21,33 @@ def destroy
     end
   end
 
+  def accept_invitation
+    if params.key?(:invitation_id)
+      redirect_to root_path unless received_invitation && token_valid? && invitation_belongs_to_current_user?
+    else
```

Comment:
if no invitation_id key this action should not be activated, so remove this `if`?